contracts/verifier/VerifierBase.sol

```diff
@@ -56,6 +56,13 @@ abstract contract VerifierBase is IVerifierBase, EIP712 {
         _cancelGroup(groupCancellation.common.account, groupCancellation.group);
     }
 
+    /// @notice Checks account authorization
+    /// @param account the account to check authorization for
+    /// @param signer the signer of the account
+    function _authorized(address account, address signer) internal virtual {
```

Comment:
it might be cleaner to have this return a `bool` instead and do the revert inside of `validateAndCancel` so that the overridden versions of this function don't have to deal with reverts as well.